base code

```diff
@@ -1,52 +1,130 @@
-board = [' ' for i in range(9)]
-icon='X'
-player = 1
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jun 19 12:04:13 2020
 
-# 9 empty values
+@author: daniveris
+"""
 
-def print_board():
+board = [' ' for i in range(16)]
+w = 'X'
+A = True
+I = [1,1,1,2,3,3,4,7]
+J = [1,3,4,3,2,3,1,1]
+winner = ''
+player = 1
+
+def play() :
+    global w
+    global A
+    global I
+    global J
+    global player
     row1= '|{}|{}|{}|'.format(board[0],board[1],board[2])
     row2= '|{}|{}|{}|'.format(board[3],board[4],board[5])
     row3= '|{}|{}|{}|'.format(board[6],board[7],board[8])
-    A = True
     print()
     print(row1)
     print(row2)
     print(row3)
     print()
-    def player_move(icon):
-        if icon == 'X':
-            player = 1
-        else:
-            player = 2
-
     print("Player {}, your turn".format(player))
     choice = int(input('Enter your move (1-9):').strip())
     if board[choice - 1] ==' ':
-        board[choice - 1] = icon
+        board[choice - 1] = w
         print()
         print(row1)
         print(row2)
         print(row3)
         print()
     else:
+            
         print()
         print('That space is taken, choose another one')
-    for i in range (0, 5):
-        for j in range(1,4):
-            if board[i] == board[i+j] and board[i+j] == board[i+2*j]:
-                A = False
-                if board[i] == 'X':
-                    winner = 1
-                else:
-                    winner = 2
-                print('The Winner is Player {}, Congratulations!'.format(winner))
-            else:
-                if icon == 'X':
-                    player_move('O')
-while True:
-    print_board()
+    #win2()
+    w = turn(w)
+    player = player_move(turn(w))
+    while A == True:
+        print_board()
 
+def turn(icon):
+    if icon == 'X':
+        icon = 'O'
+    elif icon == 'O':
+        icon = 'X'
+    return icon
 
+def player_move(icon):
+    if icon == 'X': 
+        player = 1
+    else:
+        player = 2
+    return player
```

Comment:
Instead of changing player and icon independently, the player is dependent of the icon to minimize failure.